cdk.json

```diff
@@ -0,0 +1,55 @@
+{
+  "app": "python3 app.py",
+  "watch": {
+    "include": ["**"],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "requirements*.txt",
+      "source.bat",
+      "**/__init__.py",
+      "python/__pycache__",
+      "tests"
```

Comment:
Some of these files don't exist. Was this copied from elsewhere?

Reply:
This was a hang over from running `cdk init`.... have removed.